.github/ISSUE_TEMPLATE/support_request.md

```diff
@@ -35,13 +35,27 @@ Autometa here.
 
 #### Files
 
-<!-- A list of relevant files for this request. This will help people navigate the project and offer some clues of where to start. -->
+<!-- A list of relevant files for this request. Please include descriptions of any files you have mentioned in the description, tasks/commmands or expected behavior. -->
+
+- File: Description
+- File: Description
+- File: Description
+- etc.
```

Comment:
This is still really vague. I still don't know what files you want and why. People will read this and upload their entire metagenome, or the whole of the Autometa github repository. Be specific in your request.

Reply:
Also, since you explicitly mention logs below, perhaps explain here that you don't mean logs.

Reply:
Do you think I should just remove the files section?

Reply:
I guess I just don't know what it is for. If the explanation were a bit more unambiguous it could be OK.

Reply:
I'll remove this section. I think this would probably be more relevant for a feature request rather than a support request. I was thinking along the lines of any files they may want to incorporate into the Autometa framework.